source/_patterns/02-organisms/content-headers/content-header-journal.mustache

```diff
@@ -1,136 +1,159 @@
-          </div>
-          {{#socialMediaSharers}}
-            {{> atoms-social-media-sharers-journal}}
-          {{/socialMediaSharers}}
+        <div class="content-header-grid__main">
+
+            <div class="content-header__body">
+                <h1 class="content-header__title {{#titleLength}}content-header__title--{{titleLength}}{{/titleLength}}{{^titleLength}}content-header__title--medium{{/titleLength}}">{{{title}}}</h1>
+
+                {{#impactStatement}}
+                    <div class="content-header__impact-statement">{{{impactStatement}}}</div>
+                {{/impactStatement}}
+
+            </div>
+
+            {{#authors}}
+                {{> molecules-authors}}
+            {{/authors}}
+
+            <div class="content-header__footer">
+
+                {{#isOneColumn}}
+                <div class="content-header__footer-column-left">
+                    {{#contextualData}}
+                        {{> molecules-contextual-data }}
+                    {{/contextualData}}
+                {{/isOneColumn}}
+
+                {{#meta}}
+                    <div class="content-header__meta">
+                        {{> molecules-meta-journal}}
+                    </div>
+                {{/meta}}
+
+                {{#doi}}
+                    {{> atoms-doi}}
+                {{/doi}}
+
+                {{#licence}}
+                    <ul class="content-header__icons">
+                        <li>
+                            <a href="https://en.wikipedia.org/wiki/Open_access"
+                               class="content-header__icon content-header__icon--oa">
+                                <span class="visuallyhidden">Open access</span>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="{{licence}}" class="content-header__icon content-header__icon--cc">
+                                <span class="visuallyhidden">Copyright information</span>
+                            </a>
+                        </li>
+                    </ul>
+                {{/licence}}
+                {{#isOneColumn}}
+                </div>
+                {{/isOneColumn}}
+
+                {{#isOneColumn}}
+                    <div class="content-header__one-column-container">
+                        <div data-hypothesis-trigger>
+                            <span class="content-header-grid__side-popup-block__link"
+                                  data-behaviour="HypothesisTrigger"><span
+                                    class="svg-background-image comment"></span><span class="visuallyhidden">Open annotations (there are currently <span
```

Comment:
change svg-background-image to something more agnostic as it may not be an svg in the future

Reply:
I think this is not a requirement for this ticket. It appears as a change just because Alex copied all the content from another file to this one.